Makefile

```
@@ -130,9 +130,11 @@ export XPKG_IGNORE := $(XPKG_IGNORE)
 
 CONFIG_CRD_GROUP = $(PROVIDER_NAME)
 PROVIDER_AUTH_GROUP = $(PROVIDER_NAME)
+CONFIG_DEPENDENCY_REG_ORG = $(XPKG_REG_ORGS)
```

```suggestion
CONFIG_DEPENDENCY_REG_ORG ?= $(XPKG_REG_ORGS)
```

only default if not set (e.g. via https://github.com/crossplane-contrib/provider-workflows/pull/8)

---

Done!